src/Symfony/Component/Uid/UuidV4.php

```diff
@@ -23,12 +23,19 @@ class UuidV4 extends Uuid
     public function __construct(?string $uuid = null)
     {
         if (null === $uuid) {
-            $uuid = random_bytes(16);
-            $uuid[6] = $uuid[6] & "\x0F" | "\x40";
-            $uuid[8] = $uuid[8] & "\x3F" | "\x80";
-            $uuid = bin2hex($uuid);
-
-            $this->uid = substr($uuid, 0, 8).'-'.substr($uuid, 8, 4).'-'.substr($uuid, 12, 4).'-'.substr($uuid, 16, 4).'-'.substr($uuid, 20, 12);
+            // Generate 36 random hex characters (128 bits)
```

Comment:
This comment is a lie. You are generating more than 128 bits of randomness. 18 bytes is 144 bits. Generating 128 bits was what the old code was doing.

Reply:
A _lie_  😱  //   It's fixed thank you :)